steampipecloud/table_steampipecloud_connection.go

```diff
@@ -2,10 +2,7 @@ package steampipecloud
 
 import (
 	"context"
-	"net/http"
-	"time"
 
-	"github.com/sethvargo/go-retry"
 	openapi "github.com/turbot/steampipe-cloud-sdk-go"
 
 	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
```

Comment:
Can you please add `identity_type` as an optional list key qual for this table?

Reply:
@cbruno10 list API's does not support `identity_type`. The user and org API has support for `identity_handle` only.